src/app/code/community/Omise/Gateway/etc/config.xml

```diff
@@ -74,6 +74,46 @@
                 <class>Omise_Gateway_Helper</class>
             </omise_gateway>
         </helpers>
+
```

Comment:
do we need space here?

Reply:
I was following existing conventions that were already in the `config.xml` file